Internationalization Updates (#2010)
infinitered/ignite: docs/Internationalization.md

# Internationalization in Ignite Apps

Ignite currently setup to have Internationalization setup in English, Arabic, and Korean! This is detected on app load and will set your app to that language.

> ## RTL

Reviewer: ```suggestion
## Right to Left languages (RTL)
```

Wherever we use acronyms and initialisms in the docs, the first usage in the doc should spell it out.

Since Ignite already comes with an RTL language, Arabic, adding any new ones would work by default.

### Removing RTL Support

To remove RTL support, follow the following steps:

1. `/app/i18n/i18n.ts`

- remove your RTL `.json` file from the `i18n.translations` object
- remove lines 19-21 where we allow and force RTL on the native layer

> 2. remove all other associated logic that uses export `isRTL` variable

Reviewer: ```suggestion
2. remove all other associated logic that uses the exported `isRTL` variable
```

## Adding more languages

> 1. add you language `.json` file to `app/i18n/`, i.e. `app/i18n/fr.json`

Reviewer: ```suggestion
1. add your language `.json` file to `app/i18n/`, i.e. `app/i18n/fr.json`
```

# Internationalization in Ignite Apps

Ignite currently setup to have Internationalization setup in English, Arabic, and Korean! This is detected on app load and will set your app to that language.

## Right to Left languages (RTL)

Since Ignite already comes with an RTL language, Arabic, adding any new ones would work by default.

> NOTE: if you move between a RTL and non-RTL language, you'll have to relaunch your app twice to get the correct localization logic.

Reviewer 2: Lizzi asked if this happens on both platforms or just iOS?
